input/pagecontent/change_log.md

```diff
@@ -1,3 +1,62 @@
+
+* [NeuroblastomaINSStage]
+* [NeuroblastomaINRGSSRiskAssessment]
+* [WilmsTumorStage]
+* [RhabdomyosarcomaClinicalGroupStage]
+
+### Modification of Lymphoma Stage Profile
+
+Lymphoma staging can include many modifiers, such as bulky status, extranodal involvement, splenic involvement, and symptom indicators. As a result, components for these modifiers were added to the [LymphomaStage] profile to better describe the lymphoma stage.
+
+### Addition of Risk Assessment Profiles
+
+mCODE STU3 did not include any profiles for risk assessments, so there was no indication on how to use mCODE to represent different risk assessments. To address this issue, an abstract, generic profile was created for risk assessment, and specific risk assessment profiles were added for illustrative purposes:
+
+* [CancerRiskAssessment]
+* [RhabdomyosarcomaRiskAssessment]
+* [ALLRiskAssessment]
+
+### Addition of Histology Profiles
+
+mCODE 4.0.0-ballot includes three new profiles for histologic reporting. [HistologicBehaviorAndType] reports the morphology and type of the tumor using ICD-O-3 codes and the proper suffix, while [HistologicGrade] reports the grade of the tumor. Both profiles are referenced in a summary with the [TumorMorphology] profile.
+
+* [HistologicBehaviorAndType]
+* [HistologicGrade]
+* [TumorMorphology]
+
+### Modification of Medication Profiles
+
+The normalization basis for medication dosage is important, especially for pediatric oncology. As a result, the [CancerRelatedMedicationAdministration] and [CancerRelatedMedicationRequest] profiles were modified to include an extension for the normalization basis used.
+
+### Addition of Lansky Play Performance Profile
+
+Although mCODE 3.0.0 contained two performance status profiles ([ECOGPerformanceStatus] and [KarnofskyPerformanceStatus]), it did not include the performance status most commonly used for children. As a result, the [LanskyPlayPerformanceStatus] profile was added.
+
+### Addition of Deauville Scale Profile
+
+Deauville Scale is a common assessment performed for lymphoma cancer patients, and as a result, a [DeauvilleScale] profile was added to mCODE.
+
+
+### Addition of Body Surface Area Profile
+
+Body surface area is measured/calculated commonly for pediatric patients and used to determine medication dosage. As a result, a [BodySurfaceArea] profile was added to mCODE.
+
+### Clarified Codes for Cancer Disease Status
+
+Adjusted the [ConditionStatusTrendVS] to avoid confusion. In mCODE STU3, some of the values can be true at the same time (example: a patient’s condition could have improved AND be in full remission). However, value[x] is 0.1. As a result, the codes for in full remission, partial remission, and distant metastasis present were moved into a maximum value set to assure backward compatbility with this change.
+
+
+<br/>
+<br/>
+----
+<br/>
+
+
 **The following changes occurred between the [STU 3 ballot (March 2023)] and publication of STU 3, in response to ballot comments.**
```

Comment:
I think all of the material here can be deleted.  At the top you say "go look at the change log in the relevant versions".